<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>onionSkinRendererPreferences</class>
 <widget class="QDialog" name="onionSkinRendererPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>118</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="prefs_maxBuffer_layout">
     <item>
      <widget class="QLabel" name="maxBuffer_label">
       <property name="text">
        <string>Maximum Buffer Size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="maxBuffer_spinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>200</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="prefs_relativeKeyCount_layout">
     <item>
      <widget class="QLabel" name="relativeKeyCount_label">
       <property name="text">
        <string>Relative Keys Count</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="relativeKeyCount_spinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="value">
        <number>4</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="prefs_outlineWidth">
     <item>
      <widget class="QLabel" name="outlineWidth_label">
       <property name="text">
        <string>Outline Width</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="outlineWidth_spinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>512</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="prefs_dialogButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>prefs_dialogButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>onionSkinRendererPreferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>prefs_dialogButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>onionSkinRendererPreferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
